client/document/index.jsx

```diff
@@ -198,6 +188,15 @@ class Document extends React.Component {
 						 `,
 						} }
 					/>
+					{ // Load GA only if enabled in the config.
+					appConfig( 'google_analytics_enabled' ) && (
+						<script
+							async={ true }
+							type="text/javascript"
+							src="https://www.google-analytics.com/analytics.js"
+							nonce={ analyticsScriptNonce }
+						/>
+					) }
```

Comment:
was there a merge in this PR that brought in unrelated changes?

Reply:
If you referring the above, that is done to add the `nonce` instead of whitelisting the domain `www.google-analytics.com`